bin/generateSite.php

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env php
+<?php
+
+$file_system_traversal = new \Gnorm\FileSystemTraversal();
+$repo_root = $file_system_traversal->findRepoRoot();
+$autoload = require_once $repo_root . '/vendor/autoload.php';
+
+if (!isset($autoload)) {
+    print "Unable to find autoloader for Gnorm.\n";
+    exit(1);
+}
+
+/**
+ * Options:
+ * c = The json encoded config.
+ * b = Boolean telling if this is a build.
+ */
+$options = getopt('c:b::');
```

Comment:
Same thing here.

Reply:
@ryanobrooks, the optional parameter identifier was changed to use the required format.